### PR TITLE
Update Spring Security's configuration

### PR DESCRIPTION
**Pull request by marcusdacoregio**
See:
 - [WebSecurityConfigurerAdapter deprecation](https://spring.io/blog/2022/02/21/spring-security-without-the-websecurityconfigureradapter)
 - [`authorizeHttpRequests`](https://docs.spring.io/spring-security/reference/servlet/authorization/authorize-http-requests.html)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -165,19 +165,18 @@ include::complete/src/main/java/com/example/securingweb/WebSecurityConfig.java[]
 ====
 
 The `WebSecurityConfig` class is annotated with `@EnableWebSecurity` to enable Spring
-Security's web security support and provide the Spring MVC integration. It also extends
-`WebSecurityConfigurerAdapter` and overrides a couple of its methods to set some specifics
-of the web security configuration.
+Security's web security support and provide the Spring MVC integration. It also exposes
+two beans to set some specifics of the web security configuration.
```
Comment (Buzzardo) [inline README.adoc:169]: Can you be more detailed about what specifics are exposed?

Also, "of the web security configuration" should be "for the web security configuration".

Thanks.

**Review by Buzzardo — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 1 above)
Comment (marcusdacoregio) [reply README.adoc]: > Can you be more detailed about what specifics are exposed?
It is explained in the two following paragraphs, I don't think we need to repeat it here.

**Comment by marcusdacoregio** [comment]
Thank you @Buzzardo, I've updated the PR with your suggestions

**Review by Buzzardo — APPROVED** [review]

**Comment by Buzzardo** [comment]
Thanks.

**Pull request merged**